content/best-component-library.mdx

```diff
@@ -0,0 +1,23 @@
+---
+updated: 2024-12-03
+title: "Best component library"
+authors: [PhantomKnight287]
+---
+
+There isn't a single "best" component library—it all depends on your project's requirements and the level of customization you need. For extensive customization, you might consider [ShadCN UI](https://ui.shadcn.com), which is built on Tailwind CSS and Radix UI. Alternatively, libraries like [Chakra UI](https://chakra-ui.com) and [Material UI](https://mui.com) offer pre-designed components with flexibility and accessibility in mind.
```

Comment:
Consistent capitalisation of shadcn/ui: it's stylised as "shadcn/ui" on the official website so that's the format we should use.